ocd-3093.sql

```diff
@@ -0,0 +1,22 @@
+CREATE OR REPLACE VIEW openchpl.developer_certification_body_map
```

Comment:
This isn't needed in this file. The v-next.sql file inserts the openchpl_views.sql file, so we don't need to put views (or soft-delete / grant-all statements) in ocd-nnnn files